ethdb/leveldb/leveldb.go

```diff
@@ -149,12 +150,24 @@ func NewCustom(file string, namespace string, customize func(options *opt.Option
 	return ldb, nil
 }
 
+// NewCustom returns a wrapped LevelDB object with an in-memory storage.
+func NewInMemory() (*Database, error) {
+	db, err := leveldb.Open(storage.NewMemStorage(), nil)
+	if err != nil {
+		return nil, err
+	}
+	return &Database{
+		db: db,
+	}, nil
+}
+
 // configureOptions sets some default options, then runs the provided setter.
 func configureOptions(customizeFn func(*opt.Options)) *opt.Options {
 	// Set default options
 	options := &opt.Options{
 		Filter:                 filter.NewBloomFilter(10),
 		DisableSeeksCompaction: true,
+		CompactionTableSize:    10 * opt.MiB,
```

Comment:
This value is being set here, but also being set in the persistentDb of the roundstate db. Is this one the default? Is it necessary to set it in both?

Reply:
Good point, updated.